Assets/Scripts/RedRunner/GameManager.cs

```diff
@@ -302,6 +305,25 @@ public void RespawnCharacter(Character character)
             character.Reset();
         }
 
+        public void LockCharacterToStart()
+        {
+            GameObject respawn = SpawnSingleton.instance;
+            Vector3 position = respawn.transform.position;
+            Debug.Log("Text: " + position);
+            position.y += 2.56f;
+            float width = respawn.GetComponent<SpriteRenderer>().bounds.size.x;
```

Comment:
width -> diameter for clarity if we're keeping the circle? or just a comment somewhere about it